growth/too/tasks/tiles.py

```diff
@@ -340,6 +350,9 @@ def get_planned_observations(
 
             exposure_time, field_id, prob = data[4], data[5], data[6]
 
+            if params["tilesType"] == "galaxy":
```

Comment:
For galaxy tiles, please let SQLAlchemy determine the field IDs for the PlannedObservation instances automatically.